contrib/ontologies.json

```diff
@@ -0,0 +1,302 @@
+  "result": {
+      "data": [
+          {
+            "copyright": "2000-2008 Cold Spring Harbor Laboratory and Cornell University, USA",
+            "description": "Plant Trait Ontology",
+            "documentationURL": "http://purl.obolibrary.org/obo/TO_0006061",
```

Comment:
* if we are only using plant trait ontology, this file should only have one result
* documentationURL should point to a canonical URI for the plant trait ontology
* ontologyDbId could be something like 'TO' (if this is the common identifier for all terms in the plant trait ontology). This can be any unique value, even an integer, but a name is helpful.